src/main/java/com/j256/ormlite/stmt/QueryBuilder.java

```diff
@@ -485,6 +485,10 @@ protected void appendStatementStart(StringBuilder sb, List<ArgumentHolder> argLi
 			sb.append("COUNT(").append(countOfQuery).append(") ");
 		}
 		sb.append("FROM ");
+		if (tableInfo.getSchemaName() != null && tableInfo.getSchemaName().length() > 0){
+			databaseType.appendEscapedEntityName(sb, tableInfo.getSchemaName());
+			sb.append(".");
```

Comment:
This is pedantic but please use `sb.append('.')` characters if you are adding a character.

Reply:
Ok, replaced.